var/spack/repos/builtin/packages/g2/package.py

```diff
@@ -59,6 +59,12 @@ def cmake_args(self):
 
         return args
 
+# GNU compiler is complaining if not using -fno-range-check argument
```

Comment:
Is it safe to apply this to newer gfortran versions, too? I know we need it for 9.2.0 (hera, jet) and 9.4.0 (containers), but I am not sure it's needed for gfortran-10+

Reply:
> Is it safe to apply this to newer gfortran versions, too? I know we need it for 9.2.0 (hera, jet) and 9.4.0 (containers), but I am not sure it's needed for gfortran-10+

Right, I don't know if this will affect any other machine as well. But actually, this argument is benign, it should work for any machine/compiler version. Unless we see the problem, I'd leave it as is.